kirrya95/trading-bot-dydx-deribit
# limit two instr new

- test_grid_two_instr: init, sample test works
- update_limit_order, upd_take_profit
- update limit and take profit orders work
- add KeyError check, tests work
- upd clear_level
- upd, tests work
- mv spread calcs to corresponding file in utils
- ws.close() and all tests pass
- check_total_take_profit() for two instr bot
- create_batch_limit_orders() and handling
- upd, bot creates two batch orders
- creating limit orders work, take profit orders not yet
